docs/element-definitions.md

```diff
@@ -634,34 +677,29 @@ See [Object Element](#object-element) for examples.
 
 ### Object Element
 
-Type with domain of all finite lists of _properties_.
+Type with domain of all finite sets of [_properties_](#property).
 
-Quoting from the [Element section](#element):
-> A _property_ is a tuple (`key`, `value`) where
-> - `key` SHALL be a non-empty, finite character string
-> - `value` SHALL be an _Element_
-> - two properties SHALL be equal if their keys are.
 
 #### Template
 - `element` - `"object"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `fixed` ([String](#string-element)) - SHALL restrict the domain to a specific instance of a property list. Entries in `content` of this Object Element MUST contain at least one of content, default and sample.
-    - `fixedType` ([String](#string-element)) - SHALL restrict the domain to properties listed in `content`.
+    - `fixed` ([String](#string-element)) - Restricts domain to a fixed sized set of properties typed in content, making them implicitly required. Further applies the `fixed` type attribute on items typing a single value or specifying a default value.
+    - `fixedType` ([String](#string-element)) - Restricts domain to a fixed sized set of properties, making them implicitly required.
   - `validation` - _reserved for future use_
-  - `samples` ([Array[Object]](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (Object) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - list of any of
-  - [Member](#member-element) - object property
+  - `samples` ([Array[[Object](#object-element)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
```

Comment:
Bad linking. Use something similar to `([Array](#array-element)[[String](#string-element)])`